composer.json

```diff
@@ -54,7 +54,7 @@
   },
   "scripts": {
     "post-update-cmd": [
-      "sed -i 's/final public function add_group_control/public function add_group_control/g' ./vendor/elementor/elementor/includes/base/controls-stack.php"
+      "sed -i'' 's/final public function add_group_control/public function add_group_control/g' ./vendor/elementor/elementor/includes/base/controls-stack.php"
```

Comment:
@dgwatkins can you explain this change?

Reply:
I didn't really mean to commit that change. It actually slipped in during a fixup commit.

However, the change is required to run unit tests on my local host (Mac)

Linux just takes `-I` with no arguments.
Mac takes `-I` with the extension of the bak file (ie: `.bak`).

This way of calling it `-I''` (with no spaces) plays well with both Mac & Linux versions of sed.

Reply:
Good to know!

It would be better if you could move this change in a separate commit where you explain the reason of the change.

Reply:
I see you approved, so I will make sure I issue a separate commit next time.